unit-tests/game-api-tests.js

```diff
@@ -1,32 +1,23 @@
-//describe('gameApiTest', function(){
-//
-//    var gameApi;
-//
-//    beforeEach(function(){
-//        module('tombola.noughtAndCrosses.services');
-//        inject(function($injector){
-//            gameApi = $injector.get('gameApi');
-//        });
-//    });
-//
-//    it('Should contain gameApi service', function(){
-//        expect(noughtsAndCrossesApp.gameApi)
-//            .to.equal('noughtsAndCrossesApp.service(gameApi,function ($http,$q,gameApiConstants)')
-//    });
-//
-//    it('makeMove function contains appropriate playerNumber and squareNumber', function(){
-//        expect(noughtsAndCrossesApp.gameApi.makeMove)
-//            .to.equal(gameApi.makeMove.playerNumber = 'playerNumber',gameApi.makeMove.squareNumber = 'squareNumber');
-//    });
-//
-//    it('newGame function contains constants', function(){
-//        expect(noughtsAndCrossesApp.gameApi.newGame)
-//            .to.equal(gameApi.newGame.player1 = 'player1Type',gameApi.newGame.player2 = 'player2Type');
-//    });
-//
-//    it('gameApi service contains promise', function(){
-//        expect(noughtsAndCrossesApp.gameApi)
-//            .to.equal(gameApi.callServer);
-//    });
-//
-//});
+describe('gameApiTest', function(){
+
+    var gameApi;
+    var gameModel;
+
+    beforeEach(function(){
+        module('tombola.noughtAndCrosses.services');
+        inject(function($injector){
+            gameApi = $injector.get('gameApi');
+            gameModel = $injector.get('NoughtsAndCrosses/unit-tests/mocks/mock-game-model.js')
```

Comment:
Take a look at https://docs.angularjs.org/guide/services - unit testing section at the end. Ignore how the mock is set up - just the $provide stuff in the beforeEach closure.
You should tell angular to provide the mock (not the real) object when the injection happens. To do this you use the $provide to override the default real object with the mock.